src/classes/RD_AddDonationsBTN_CTRL.cls

```diff
@@ -53,18 +53,22 @@ public with sharing class RD_AddDonationsBTN_CTRL {
     public PageReference buttonClick(){
         Savepoint sp = Database.setSavepoint();
         try {
+
```

Comment:
@mpusto - I think as we work through the security changes, we should try to be consistent in the appearance of the messages. I like the softer touch of the info style warning on this PR but the last payment warning we issued an error toast. Attaching a screen shot of both:
![screen shot 2018-10-15 at 11 38 57 am](https://user-images.githubusercontent.com/450473/46964902-ef9fb000-d06e-11e8-8e4d-47357722516b.png)
![screen shot 2018-10-15 at 11 34 52 am](https://user-images.githubusercontent.com/450473/46964903-ef9fb000-d06e-11e8-9133-6eb8955f15d3.png)

@dvenor - Either way, I think we should stick with the page header and eliminate the need for creating a separate button to return to the recurring donation record.

Reply:
We should probably use the illustration pattern in Lightning Design System for access errors: https://lightningdesignsystem.com/components/illustration/#No-Access

Though it would be better if they don't see the button to create the record in the first place (recognizing the platform doesn't necessarily allow us to set perms on a custom button based on profile etc.)

Reply:
I went with the same look as Jenny's payment example, except displayed the message as info instead of an error.